app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -303,4 +303,15 @@
   <dimen name="app_version_image_view_margin_start">24dp</dimen>
   <dimen name="app_version_text_view_margin_end">28dp</dimen>
 
+  <!-- Completed Story Item -->
+  <dimen name="completed_story_item_topic_container_marginStart">24dp</dimen>
+  <dimen name="completed_story_item_topic_container_marginEnd">24dp</dimen>
+  <dimen name="completed_story_item_topic_container_marginTop">36dp</dimen>
```

Comment:
Please add bottom margin as `0dp` otherwise it will end up using value from `layout/dimens.xml` file which is `12dp`.